util/ingestor.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+# ================================================================
+# A simple driver for ingestion of anndata to a TileDB group.
+#
+# * Invoke this with no arguments to process:
+#   o canonical input:  ./anndata/pbmc3k_processed.h5ad
+#   o canonical output: ./tiledb-data/pbmc3k_processed/
+#
+# * Invoke this with one argument /path/to/some/somename.h5ad:
+#   o Output will be ./tiledb-data/somename
+#
+# * Invoke this with two arguments to specify input anndata HD5 file
+#   and output TileDB group.
+#
+# Nominal immediate-term support is to local disk, although output to tiledb:/...
+# URIs will be supported.
+#
+# Note this removes and recreates the destination TileDB group on each invocation.
+# ================================================================
+
+from mSCGroup import SCGroup
+import sys, os, shutil
+
+input_path  = 'anndata/pbmc3k_processed.h5ad'
+output_path = 'tiledb-data/pbmc3k_processed'
+if len(sys.argv) == 3:
```

Comment:
Not a necessary change now but can replace with `argparse` module in the future.